1-exercises/A-accessing-values/exercise2.js

```diff
@@ -17,7 +17,7 @@ let capitalCities = {
 */
 
 let myCountry = "UnitedKingdom";
-let myCapitalCity; // complete the code
+let myCapitalCity = capitalCities["UnitedKingdom"]; // complete the code
```

Comment:
this works but you have the `myCountry` variable available to you, so this can be used like so `capitalCities[myCountry]`